shipyard-controller/debug-ui/fetch_events.js

```diff
@@ -0,0 +1,75 @@
+fetchEvents();
```

Comment:
please, align this across files. I would suggest to don't call functions directly from the js files.